src/components/Explore/utils.ts

```diff
@@ -77,13 +79,13 @@ export const exploreMetricData: {
     series: [
       {
         label: 'Cases',
-        tooltipLabel: 'Confirmed cases',
+        tooltipLabel: '7-day avg. cases',
         datasetId: 'rawDailyCases',
         type: ChartType.BAR,
       },
       {
         label: '7 Day Average',
-        tooltipLabel: 'Confirmed cases',
+        tooltipLabel: '7-day avg. cases',
```

Comment:
We need to say "7-day avg. deaths" etc. for the other metrics, right?

Reply:
yes, will fix soon!